recording_server/src/utils/Logger.ts

```
@@ -111,160 +108,149 @@ export function setupConsoleLogger() {
-        logger.error(`Failed to upload ${options.type} logs to S3:`, error)
-        throw error
+        logger.error(`Failed to upload ${options.type} logs to S3:`, error);
+        throw error;
     }
 }
 
 export function setupExitHandler() {
     process.on('uncaughtException', async (error) => {
-        logger.error('Uncaught Exception: ' + error)
+        logger.error('Uncaught Exception: ' + error);
         try {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)